draft-thomson-webpush-http2.xml

```diff
@@ -109,6 +107,56 @@
         overhead costs.
       </t>
       <t>
+        A push server that does not support reliable delivery over intermittent
+        network connections or failing applications on devices, forces the
+        device to acknowledge receipt directly to the application server,
+        incurring additional power drain in order to establish (usually secure)
+        connections to the individual application servers.
+      </t>
+      <t>
+        While reliability is not required for messages that expire in few
+        seconds (e.g. an incoming call) or collapsible ones (e.g. the current
+        number of unread emails), it is more important when messages contain
+        information that is longer lasting, e.g. a command to update a
+        configuration value, or the acknowledgement of a financial transaction
+        or workflow step. In particular, in the case of power-constrained
+        devices, it is preferable to transmit the actual information in the
+        “pushed” message reliably, instead of forcing them to reconnect
+        periodically to get the current state.
+      </t>
+      <t>
+        An open standard to "push" messages to embedded and mobile devices:
+        <list style="symbols">
+          <t>
+            Simplifies deployment of "push" features across a variety of mobile
+            and embedded device platforms
+          </t>
+          <t>
+            Creates an ecosystem of services (e.g. consolidation services) and
+            development tools enabling efficient "push"
+          </t>
+          <t>
+            Technically enables consolidating real-time events into a single
+            session which is impossible when each "push" implementation is built
+            in isolation
+          </t>
+        </list>
+      </t>
+      <t>
+        There are two primary scenarios under consideration:
+        <list style="symbols">
+          <t>
+            Web applications in a mobile user agent and
+          </t>
+          <t>
+            Embedded devices receiving push messages from cloud services through
+            an intermediate “field gateway”, i.e. a reasonably powerful device
+            (capable of secure HTTP/2 communications), which acts as a local
+            agent.
+          </t>
+        </list>
+      </t>
```

Comment:
I don't see any value in the two lists above here.  As I noted above, I'd prefer to have mention of specific deployment scenarios removed.